### PR TITLE
add databend bugs

### PR DESCRIPTION
**Pull request by hanyisong**

### CODE REVIEW

File: src/sqlancer/databend/DatabendErrors.java [hunk 1]
```diff
@@ -30,7 +30,7 @@ public static void addExpressionErrors(ExpectedErrors errors) {
         if (DatabendBugs.bug9264) {
             errors.add("assertion failed: offset + length <= self.length");
         }
-        errors.add("validity's length must be equal"); // bug
+        // errors.add("validity's length must be equal"); // bug
```
Comment (mrigger) [inline src/sqlancer/databend/DatabendErrors.java:33]: Perhaps we can remove this comment instead?

Comment (hanyisong) [reply src/sqlancer/databend/DatabendErrors.java:33]: This error was triggered with a small probability before, I forgot to leave a log.

Comment (mrigger) [reply src/sqlancer/databend/DatabendErrors.java:33]: Okay, I see. Then let's leave the comment in for now.

**Comment by mrigger** [comment]
By the way, before reporting bugs, it is useful to reduce the bug-inducing test case to a minimal version that still triggers the bug (e.g., by repeatedly removing statements as long as the error/incorrect result is triggered). This also helps in identifying what potential features trigger the bug, whose generation can be disabled until the bug is fixed.

**Comment by mrigger** [comment]
@kshehata recently added experimental support for test case reduction (see https://github.com/sqlancer/sqlancer/pull/624). Perhaps you could check whether this is useful for your efforts in testing Databend?

**Comment by hanyisong** [comment]
Thanks for the advice, I was too anxious.

**Pull request merged**